src/init.cpp

```diff
@@ -1628,6 +1643,17 @@ bool AppInit2(boost::thread_group& threadGroup, CScheduler& scheduler)
 
```

Comment:
Why is this change missing?? https://github.com/bitcoin/bitcoin/pull/8392/files#diff-c865a8939105e6350a50af02766291b7R1304

Reply:
Because this is only useful to fix missing segwit data https://github.com/sipa/bitcoin/commit/6032f6930a56c107dad8f30c05fec4aab79c8c22 which we don't have anyway